diff --git a/CMakeLists.txt b/CMakeLists.txt
index 769f2bdc1..39c7301dd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -310,7 +310,7 @@ endif()
 
 # SUNDIALS
 if (MFEM_USE_SUNDIALS)
-  set(SUNDIALS_COMPONENTS CVODES ARKODE KINSOL NVector_Serial)
+  set(SUNDIALS_COMPONENTS cvodes arkode kinsol nvecserial)
   if (MFEM_USE_MPI)
     list(APPEND SUNDIALS_COMPONENTS NVector_Parallel NVector_MPIPlusX)
   endif()
@@ -321,6 +321,11 @@ if (MFEM_USE_SUNDIALS)
     list(APPEND SUNDIALS_COMPONENTS NVector_Hip)
   endif()
   find_package(SUNDIALS REQUIRED ${SUNDIALS_COMPONENTS})
+  set(SUNDIALS_FOUND TRUE)
+  set(SUNDIALS_LIBRARIES)
+  foreach(COMP IN LISTS SUNDIALS_COMPONENTS)
+      list(APPEND SUNDIALS_LIBRARIES "SUNDIALS::${COMP}")
+  endforeach()
 endif()
 
 # SuperLU_DIST can only be enabled in parallel
