diff --git a/CMakeLists.txt b/CMakeLists.txt
index 769f2bdc1..2ba963e97 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -274,6 +274,7 @@ endif()
 
 if (MFEM_USE_METIS)
   find_package(METIS REQUIRED)
+  set(METIS_LIBRARIES metis)
 endif()
 
 if (MFEM_USE_GINKGO)
@@ -363,7 +364,8 @@ endif()
 
 # NetCDF
 if (MFEM_USE_NETCDF)
-  find_package(NetCDF REQUIRED)
+  find_package(netCDF REQUIRED)
+  set(NETCDF_LIBRARIES ${netCDF_LIBRARIES})
 endif()
 
 # MPFR
