diff --git a/CMakeLists.txt b/CMakeLists.txt
index 769f2bdc1..ddcf0d957 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -310,7 +310,7 @@ endif()
 
 # SUNDIALS
 if (MFEM_USE_SUNDIALS)
-  set(SUNDIALS_COMPONENTS CVODES ARKODE KINSOL NVector_Serial)
+  set(SUNDIALS_COMPONENTS cvodes arkode kinsol nvecserial)
   if (MFEM_USE_MPI)
     list(APPEND SUNDIALS_COMPONENTS NVector_Parallel NVector_MPIPlusX)
   endif()
