diff --git a/CMakeLists.txt b/CMakeLists.txt
index 769f2bdc1..e03e8ebc6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -274,6 +274,7 @@ endif()
 
 if (MFEM_USE_METIS)
   find_package(METIS REQUIRED)
+  set(METIS_LIBRARIES metis)
 endif()
 
 if (MFEM_USE_GINKGO)
@@ -364,11 +365,17 @@ endif()
 # NetCDF
 if (MFEM_USE_NETCDF)
   find_package(NetCDF REQUIRED)
+    find_package(netCDF REQUIRED)
+  set(NETCDF_FOUND TRUE)
+    set(NETCDF_LIBRARIES netCDF::netcdf)
 endif()
 
 # MPFR
 if (MFEM_USE_MPFR)
-  find_package(MPFR REQUIRED)
+  find_package(PkgConfig)
+  pkg_check_modules(mpfr REQUIRED IMPORTED_TARGET mpfr)
+  set(MPFR_FOUND TRUE)
+  set(MPFR_LIBRARIES PkgConfig::mpfr)
 endif()
 
 if (MFEM_USE_CEED)
