diff --git a/config/cmake/config.hpp.in b/config/cmake/config.hpp.in
index 7cb2a958d..bac251b5d 100644
--- a/config/cmake/config.hpp.in
+++ b/config/cmake/config.hpp.in
@@ -30,12 +30,6 @@
 #define MFEM_VERSION_MINOR (((MFEM_VERSION)/100)%100)
 #define MFEM_VERSION_PATCH ((MFEM_VERSION)%100)
 
-// MFEM source directory.
-#define MFEM_SOURCE_DIR "@MFEM_SOURCE_DIR@"
-
-// MFEM install directory.
-#define MFEM_INSTALL_DIR "@MFEM_INSTALL_DIR@"
-
 // Description of the git commit used to build MFEM.
 #cmakedefine MFEM_GIT_STRING "@MFEM_GIT_STRING@"
 
